### PR TITLE
contact page for user

### PR DESCRIPTION
**Pull request by at-trinhle**

### CODE REVIEW

File: final-app/src/app/contact/contact.component.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<section class="user-contact">
+  <div class="banner-contact">
+    <h2 class="title-contact">contact us</h2>
+  </div>
+</section>
+<section class="map">
```
Comment (at-dongho) [inline final-app/src/app/contact/contact.component.html]: - Change class to have the prefix `section-` and remember using for all section
- Ex: `section-map`

File: final-app/src/styles/pages/_contact.scss [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      position: absolute;
+      top: 0;
+      width: 100%;
+    }
+  }
+}
+.contact-form {
+  text-align: left;
+  position: absolute;
+  top: -140px;
```
Comment (at-dongho) [inline final-app/src/styles/pages/_contact.scss]: Why do you using `top: -140px; right: -338px;` for this.

File: final-app/src/styles/pages/_contact.scss [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+.map {
+  margin: 150px 0 100px;
+  text-align: center;
+  position: relative;
+}
+.section-social {
+  position: relative;
+  background-image: url("../assets/images/banner_contact1.png");
+  padding: 200px 0;
+  &:after {
```
Comment (at-dongho) [inline final-app/src/styles/pages/_contact.scss]: Using 2 colons for `::after, ::before`

**Comment by at-trinhle** [comment]
merged contact-page to master

**Pull request merged**